Doyeon-k-fanshawe/group2_bootcamp
# Adding image to page

## What does the PR do?
* It adds profile image to index.html
## How do you check this PR?
* Pull down the branch
*  See added image

## Screenshots 
None.
